airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -1650,6 +1655,26 @@ definitions:
       type:
         type: string
         enum: [JsonDecoder]
+  JsonlDecoder:
+    title: Jsonl Decoder
```

```suggestion
    title: JSONL Decoder
```

I'm not certain, but felt that acronyms should be capital letters. If we don't do this in other spots, feel free to ignore this.